apps/api/src/app/workflows-v2/usecases/build-step-issues/build-step-issues.usecase.ts

```diff
@@ -257,4 +276,27 @@ export class BuildStepIssuesUsecase {
 
     return error.message || 'Invalid value';
   }
+
+  @Instrument()
+  private validateSkipField(skipLogic: RulesLogic<AdditionalOperation>): StepIssuesDto {
+    const issues: StepIssuesDto = {};
+
+    const queryValidatorService = new QueryValidatorService();
+    const skipRulesIssues = queryValidatorService.validateQueryRules(skipLogic);
+
+    if (skipRulesIssues.length > 0) {
+      issues.controls = {
+        skip: skipRulesIssues.map((issue) => ({
+          issueType:
+            issue.type === QueryIssueTypeEnum.MISSING_VALUE
+              ? StepContentIssueEnum.MISSING_VALUE
+              : StepContentIssueEnum.ILLEGAL_VARIABLE_IN_CONTROL_VALUE,
+          message: issue.message,
+          variableName: issue.path.join('.'),
```

Comment:
I do use this field as a `path` to the variable in the conditions editor form. The path is represented by the array with values like `[0, 1]` which means: group on level 0, and rule with index 1.